swift-spaces-real-time-app/MuxSpacesUIKit/View/JoinSpaceViewController.swift

```diff
@@ -0,0 +1,548 @@
+//
+//  JoinSpaceViewController.swift
+//  
+
+import Combine
+import UIKit
+
+import MuxSpaces
+
+class JoinSpaceViewController: UIViewController {
+
+    var controlsStackView: UIStackView = UIStackView()
+
+    var joinSpaceButton: UIButton = UIButton(
+        configuration: .filled()
+    )
+
+    var audioLabel: UILabel = UILabel()
+    var audioToggle: UISwitch = UISwitch()
+    var audioStackView: UIStackView = UIStackView()
+
+    var audioOptionsLabel: UILabel = UILabel()
+    var audioOptionsStackView: UIStackView = UIStackView()
+
+    var targetActions: [TargetActionSender] = []
+
+    var videoLabel: UILabel = UILabel()
+    var videoToggle: UISwitch = UISwitch()
+    var videoStackView: UIStackView = UIStackView()
+
+    var captureDeviceSelectionLabel: UILabel = UILabel()
+    var captureDeviceSelectionControl: UISegmentedControl = UISegmentedControl()
+
+    var cancellables: [AnyCancellable] = []
+
+    @Published var isAudioPublishingEnabled: Bool = false
+    @Published var isVideoPublishingEnabled: Bool = false
```

Comment:
```suggestion
    @Published var isAudioPublishingEnabled: Bool = true
    @Published var isVideoPublishingEnabled: Bool = true
```

Reply:
I think it makes more sense to default this to enabled so audio and video are published on joining.